src/frontend/apps/desk/src/components/Text.tsx

```diff
@@ -33,10 +39,12 @@ export interface TextProps extends BoxProps {
 
 export const TextStyled = styled(Box)<TextProps>`
   ${({ $weight }) => $weight && `font-weight: ${$weight};`}
-  ${({ $size }) => $size && `font-size: ${$size};`}
-  ${({ $color }) => $color && `color: ${$color};`}
+  ${({ $size }) =>
+    $size &&
+    `font-size: ${$size in sizes ? sizes[$size as TextSizes] : $size};`}
   ${({ $theme, $variation }) =>
     `color: var(--c--theme--colors--${$theme}-${$variation});`}
+  ${({ $color }) => $color && `color: ${$color};`}
```

Comment:
nothing has changed for $color right? Did I miss something?

Reply:
The color prop didn't work because it was overridden by the props theme and variation, like that it works.